wrappers/python/xraydb/xraydb.py

```diff
@@ -221,26 +238,56 @@ def query(self, *args, **kws):
         "generic query"
         return self.session.query(*args, **kws)
 
+    def get_version(self, long=False, with_history=False):
+        """
+        return sqlite3 database and python library version numbers
+
+        Parameters:
+            long (bool): show timestamp and notes of latest version [False]
+            with_history (bool): show complete version history [False]
+
+        Returns:
+            string: version information
+        """
+        out = []
+        rows = self.tables['Version'].select().execute().fetchall()
+        if not with_history:
+            rows = rows[-1:]
+        if long or with_history:
+            for row in rows:
+                out.append("XrayDB Version: %s [%s] '%s'" % (row.tag,
+                                                             row.date,
+                                                             row.notes))
+            out.append("Python Version: %s" % __version__)
+            return "\n".join(out)
+        else:
+            return "XrayDB Version: %s, Python Version: %s" % (rows[0].tag,
+                                                               __version__)
+
+
     def f0_ions(self, element=None):
         """
-        return list of ion names supported for the .f0() calculation
-        from Waasmaier and Kirfel
+        return list of ion names supported for the .f0() function.
+
 
         Parameters:
             element (string, int, pr None):  atomic number, symbol, or ionic symbol
                     of scattering element.
 
         Returns:
-            if element is None, all 211 ions are returned.
-            if element is not None, the ions for that element are returned
+            list:  if element is None, all 211 ions are returned.
+                   if element is not None, the ions for that element are returned
 
         Example:
-             >>> xdb = XrayDB()
-             >>> xdb.f0_ions('Fe')
-             ['Fe', 'Fe2+', 'Fe3+']
+            >>> xdb = XrayDB()
+            >>> xdb.f0_ions('Fe')
+            ['Fe', 'Fe2+', 'Fe3+']
 
         Notes:
-           Z values from 1 to 98 (and symbols 'H' to 'Cf') are supported.
+            Z values from 1 to 98 (and symbols 'H' to 'Cf') are supported.
+
+        References:
+            Waasmaier and Kirfel
```

Comment:
It will be much better if reference can be something like
```
.. [1] O. McNoleg, "The integration of GIS, remote sensing,
   expert systems and adaptive co-kriging for environmental habitat
   modelling of the Highland Haggis using object-oriented, fuzzy-logic
   and neural-network techniques," Computers & Geosciences, vol. 22,
   pp. 585-588, 1996.
``` 

https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt